tier2/hooks/post_gen_project.py

```diff
@@ -30,8 +31,52 @@ def addTopic():
     ]
     subprocess.call(gh_cli_command)
 
+def addMaintainer():
+    maintainers = []
+    add_maintainer = True
+    while add_maintainer:
+        maintainer = {}
+        maintainer["role"] = input("Maintainer's Role (Reviewer, Approver, Maintainer): ").strip()
+        maintainer["name"] = input("Maintainer's Name: ").strip()
+        maintainer["github_username"] = input("Maintainer's GitHub Username: ").strip()
```

Comment:
For GitHub username, could we prepend a "@" right before the username (e.g. @natalialuzuriaga, @creativenick ) since on GH this would create a link to the user's profile. Please add this change to all tiers

Reply:
My bad, I saw you did this for Tier 3/4! Could we do this for 2